python/subscribe/subscribe.py

```diff
@@ -67,36 +83,20 @@ def main():
     )
 
     parser.add_argument(
-        "-n",
-        "--namespace",
-        help="Namespace to select frames on",
-        type=str,
-        required=True,
-        nargs="*"
-    )
-
-    parser.add_argument(
-        "-s",
-        "--signal",
-        help="Signal to subscribe to",
-        required=True,
-        nargs="*"
+        "-s", "--signals", help="Signal to subscribe to", required=True, nargs="*"
```

Comment:
Should we add an example? <namespace>:<signal name>

Reply:
You mean in the help text?

Reply:
That was my thought but I'm not sure